docs/configuration/itools.md

```diff
@@ -47,6 +39,12 @@ The `itools_config_dir` property is deprecated since V2.2.0. Use the `powsybl_co
 ## itools_config_name
 The `itools_config_name` property is deprecated since V2.2.0. Use the `powsybl_config_name` property instead. 
 
+## mpi_hosts
```

Comment:
Maybe add "deprecated" in the title to make it even more clear ?

Reply:
The title of this section is _Deprecated properties_. It's strange to repeat it everywhere. I use this convention also in configuration modules.

Reply:
You're right, I have missed it in the diff.